client/components/common/NavBar.js

```diff
@@ -38,9 +43,16 @@ const NavBar = (props) => {
     <Fragment>
       <nav className="top-navbar">
         <div className="navbar-header">
-          <button
+          <div>
+            <button type="button" onClick={openNav} className="navbar-brand-mobile">
+              {LogoReplacement}
+            </button>
+          </div>
+          {/* Supposed to be a navbar */}
+          {/* <button
```

Comment:
Hmmm .... Why is this commented out ?

Reply:
I assume the dev that was working on the navBar earlier had plans of creating a sideNav for mobile screens but probably couldn't get to it. Deleting the comment now.